apps/web/cypress/tests/organization-page.spec.ts

```diff
@@ -1,5 +1,6 @@
 describe('Organization Page', function () {
   beforeEach(function () {
+    cy.mockFeatureFlags({ IS_INFORMATION_ARCHITECTURE_ENABLED: true });
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Good catch!‏